opengrok-web/src/main/java/org/opengrok/web/api/v1/controller/ConfigurationController.java

```diff
@@ -91,4 +94,37 @@ public void reloadAuthorization() {
         env.getAuthorizationFramework().reload();
     }
 
+    private Object getConfigurationValueException(String fieldName) throws IOException {
+        final IOException[] capture = new IOException[1];
+        final int EXCEPTION_INDEX = 0;
+        Object result = env.syncReadConfiguration(configuration -> {
+            try {
+                return ClassUtil.getFieldValue(configuration, fieldName);
+            } catch (IOException ex) {
+                capture[EXCEPTION_INDEX] = ex;
+                return null;
+            }
+        });
+        if (capture[EXCEPTION_INDEX] != null) {
+            throw capture[EXCEPTION_INDEX];
```

Comment:
the exception here is only translated to `throw new WebApplicationException(e, Response.Status.BAD_REQUEST)`

can you thro that one here and remove the `try` block in the caller?

Reply:
Yes